### DISCUSSION TITLE
Git archives

### DISCUSSION THREAD

**Discussion by tabascounlimitedoutsell** [dandykong/nopssdk#3] [Q&A] [answered]
Anyone would not have happened to have cloned the original repository from iArtorias before it was removed? :-)

**Comment by dandykong** [comment, marked as answer]
This is the closest thing we have to a clone right now, I had to pull it off the Internet Archive and reupload it manually.

**Comment by dandykong** [comment]
I just turned on discussions, so no need to open issues for questions anymore. Moving to Q&A.
